agent/tests/test_user_instrument.c

```diff
@@ -97,7 +97,7 @@ static void test_hashmap_wraprec() {
 
   /* instrument user function */
   user_func1_wraprec = nr_php_add_custom_tracer_named(
-      user_func1_name, nr_strlen(user_func1_name));
+      user_func1_name, nr_strlen(user_func1_name), false /*is_transient*/ );
```

Comment:
Does this need updating with the enum?

Reply:
good catch- still passed the test because false was cast to the NOT_TRANSIENT enum. fixed in [59e20f4](https://github.com/newrelic/newrelic-php-agent/pull/679/commits/59e20f46755defa8283848df0ecc1dd36804df09)